apps/sms/test/marionette/drafts_test.js

```diff
@@ -191,5 +191,19 @@ marionette('Messages Drafts', function() {
 
       assertDraft(draft);
     });
+
+    test('Thread data is cleaned after leaving conversation' +
```

Comment:
__question:__ I'm doing (well, have already finished) big rewrite of drafts tests (more tests + new view/accessors paradigm) in https://github.com/mozilla-b2g/gaia/pull/30898 that covers this use case as well. So would you mind to not do this test in your PR? Otherwise I'll have more work during rebase with the same outcome anyway :)